docs/triggers/trigger-generic-inputs.md

```diff
@@ -0,0 +1,80 @@
+# Trigger Creation Workflow Definition Inputs
```

Comment:
```suggestion
# Trigger Generics
```

This isn't specific to Trigger Creation or to the `inputs` property. I think the term "definition input" is confusing here since triggers accept `inputs` and that's not what we should be referring to.